src/ansys/pytwin/examples/downloads.py

```diff
@@ -111,24 +112,26 @@ def download_file(
     file_name: str, directory: str, force_download: Optional[bool] = False, destination: Optional[str] = None
 ):
     """
-    Download an example data file.
-    Examples files are downloaded to a persistent cache to avoid downloading the same file twice.
+    Download the input file for an example.
```

Comment:
@PipKat This method can be used to download any files that is used in one of the documented example. It could an input file in csv format, the twin model file in .twin format or any other useful files for a particular example. So, I am not sure it makes sense to talk about "the input file" anywhere in the doc string.
My suggestion would be to say here: "Download a file that is used for an example." and the replace "input file" with "example file" when meaningful. 
What do you think? Thanks

Reply:
Here is my suggestion:
"""
Download a file that is used for a PyTwin example.

The files are downloaded from the PyTwin example files repository whose URL is given by the
``pytwin.examples.downloads.EXAMPLES_REPO`` constant. All example files are downloaded to a persistent cache to
avoid downloading the same file twice.

Parameters
----------
file_name : str
  Name of the example file.
directory : str
  Path to the directory in the example files repository where to find the example file.
force_download : bool, optional
  Whether to force deletion of an example file so that it can be downloaded again. The default is ``False``.
destination : str, optional
  Path to download the example file to. The default is ``None``, in which case the example file is
  downloaded to the user's temporary folder.

Returns
-------
str
  Path to the downloaded example file.

Examples
--------
>>> from pytwin import download_file
>>> path = download_file("CoupledClutches_23R1_other.twin", "twin_files", force_download=True)
"""